src/Common/Http/HttpRestRouteHandler.php

```diff
@@ -14,10 +14,22 @@
 
 namespace OpenEMR\Common\Http;
 
+use OpenEMR\Common\Logging\SystemLogger;
+
 class HttpRestRouteHandler
 {
     public static function dispatch(&$routes, $route, $request_method, $return_method = 'standard')
     {
+        // this is already handled somewhere else.
+        // let's quickly be able to enable our CORS at the PHP level.
+        header("Access-Control-Allow-Credentials: true");
+        header("Access-Control-Allow-Headers: origin, authorization, accept, content-type, x-requested-with");
+        header("Access-Control-Allow-Methods: GET, HEAD, POST, PUT, DELETE, TRACE, OPTIONS");
+        header("Access-Control-Allow-Origin: *");
```

Comment:
do we need here
header("Cache-Control": "no-store");       
header("Pragma": "no-cache");

Reply:
@sjpadgett Is that part of the FHIR spec? I would think with an Authorization: Bearer token we wouldn't want to cache anything so this might be a good place for it, but I don't know if we are using that header anywhere else?

Reply:
I do on most responses in grant flows and it is actually an ONC requirement on many responses. On resource server i'm not sure doubtful. We can leave out for now.